README.md

```diff
@@ -175,6 +175,31 @@ The default behavior is to list Web Archive files in the `archives` directory. W
 
 </details>
 
+<details>
+<summary>
+
+#### `runtimeOnlyArchives`
+
+Configure location of web archives when the website loads in the browser.
+
+</summary>
+
+| Key                   | Default Value | Value Type |                                                        |
+| --------------------- | ------------- | ---------- | ------------------------------------------------------ |
+| `runtimeOnlyArchives` | `undefined`   | `string`   | Path to JSON file with `archives`, relative to `_site` |
+
+By default, the generator configures the location of your archives at build-time (i.e. when you run `npm run build`) in order to pre-process data and render static HTML based on that data. However, you may have a use case where you need to check archive locations every time the website loads in the browser. Setting `runtimeOnlyArchives` enables you to do things like configure and update archive data without redeploying your entire website.
+
+Caveats:
+
+- The generated sitemap will no longer list a page per archive.
+- Top-level `archives` config must be a JSON file relative to the generated `_site` directory. For example, if you have a JSON file in `_site/data/archive-list.json`, `archives` should be `./data/archive-list.json`.
```

Comment:
This means that any local files listed in `archives` config in `./data-archive-list.json` should be relative to `_site`, right?

Reply:
Yep, I'll clarify